src/components/AuthoritiesLookup/AuthoritiesLookup.js

```diff
@@ -35,11 +37,15 @@ const propTypes = {
   isLoading: PropTypes.bool.isRequired,
   onNeedMoreData: PropTypes.func.isRequired,
   onSubmitSearch: PropTypes.func.isRequired,
-  query: PropTypes.string.isRequired,
+  query: PropTypes.string,
   searchQuery: PropTypes.string.isRequired,
   totalRecords: PropTypes.number.isRequired,
 };
 
+const excludedFilters = new Set([
+  FILTERS.REFERENCES,
```

Comment:
If `FILTERS.REFERENCES` is an array, do we need to spread it with `...` operator inside of `Set`?

Reply:
Oh, seems like its not